From 7c8144541e9370aa7bd54fc65f971143db964143 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 31 May 2023 10:12:06 -0700
Subject: [PATCH 09/11] Disable protonmsg plugin tests

We will not be using this plugin for Clear Linux OS, so no need to run
the associated tests.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 requirements.txt                                                | 2 --
 .../{test_protonmsg.py => test_protonmsg.py.do-not-test}        | 0
 2 files changed, 2 deletions(-)
 rename tests/test_plugins/{test_protonmsg.py => test_protonmsg.py.do-not-test} (100%)

diff --git a/requirements.txt b/requirements.txt
index 4c09d14..d4570f3 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -3,6 +3,4 @@ Cheetah;python_version < '3.0'
 Cheetah3;python_version >= '3.0'
 psycopg2;python_version >= '3.0'
 python-multilib
-python-qpid-proton<0.38.0;python_version < '3.0'
-python-qpid-proton;python_version >= '3.0'
 rpm-py-installer
diff --git a/tests/test_plugins/test_protonmsg.py b/tests/test_plugins/test_protonmsg.py.do-not-test
similarity index 100%
rename from tests/test_plugins/test_protonmsg.py
rename to tests/test_plugins/test_protonmsg.py.do-not-test
-- 
2.40.1

